js/app/src/components/File/File.svelte

```diff
@@ -80,10 +81,11 @@
 									file_data.orig_name = file_data.name;
 									file_data.name = response.files[i];
 									file_data.is_file = true;
+									file_data.blob = null;
```

Comment:
Think we just need to use `undefined` for all of these empty blob values and those errors.

Reply:
beat me to it.